dashboard/common/utils.py

```diff
@@ -66,7 +67,7 @@ def get_myla_globals(current_user):
     logout_url = ""
     google_analytics_id = ""
 
-    is_superuser = current_user.is_superuser
+    is_superuser = current_user.is_staff
```

Comment:
Nit-pick: what inspired this change? Kinda weird to me that we're still calling it `is_superuser`.